src/Interpreters/ActionsDAG.cpp

```diff
@@ -238,6 +235,86 @@ const ActionsDAG::Node & ActionsDAG::addFunction(
     return addNode(std::move(node));
 }
 
+const ActionsDAG::Node & ActionsDAG::addFunction(
```

Comment:
I think you forgot to remove copy-paste

Reply:
I decided to have 2 overloads for adding functions to the `ActionsDAG` to make a smooth change.